tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -97,7 +97,6 @@
     "azure-mixedreality-authentication",
     "azure-ai-ml",
     "azure-iot-modelsrepository",
-    "azure-monitor-ingestion",
```

Comment:
This is not blocking the PR, but I know @scbedd was working on translating these opt-out lists to pyproject.toml, but I'm not sure the status. Scott - should we continue to use this exclusion list or add pyproject.toml to our libraries?